packages/SwingSet/src/kernel/kernelQueue.js

```diff
@@ -82,19 +82,18 @@ export function makeKernelQueueHandler(tools) {
   }
 
   /**
-   * Enqueue a message to some kernel object, as if the message had been sent
-   * by some other vat. This requires a kref as a target.
+   * Enqueue a message to some kernel object, as if the message had been sent by
+   * some other vat. This requires a kref as a target.
    *
-   * @param {string} kref  Target of the message
-   * @param {string} method  The message verb
-   * @param {*} args  The message arguments
-   * @param {ResolutionPolicy=} policy How the kernel should handle an eventual
-   *    resolution or rejection of the message's result promise. Should be
-   *    one of 'none' (don't even create a result promise), 'ignore' (do
-   *    nothing), 'logAlways' (log the resolution or rejection), 'logFailure'
-   *    (log only rejections), or 'panic' (panic the kernel upon a
-   *    rejection).
-   * @returns {string | undefined} the kpid of the sent message's result promise, if any
+   * @param {string} kref Target of the message
```

Comment:
There are a gajillion instances of this, but just commenting on one as an exemplar: The two spaces between the parameter name and the text describing it is deliberate formatting and should not be squeezed out (though punctuation of some sort would be an acceptable alternative)

Reply:
Using two spaces is not JSDoc.

> though punctuation of some sort would be an acceptable alternative

Hyphen as an optional separator for params is part of JSDoc. https://jsdoc.app/tags-param.html

Reply:
I have personally never seen or used the 2 spaces. I have seen the optional `-` separator, but I've also noticed some tools with subpar support for it

Reply:
This is a convention we adopted at Electric Communities when JavaDoc (the ur-parent of JSDoc) first came on the scene.  The doc formatter can render the parameter name in a different typeface or color, making it easy to visually separate the name from the text that follows it, but when you are looking at the code itself it helps to have some kind of demarcation and this one is very easy to do.

In this case it shouldn't matter that the two spaces thing is not part of JSDoc, because in the context where it matters we're not looking at what JSDoc interpreted (which in this case would be fine) but at what text is actually in the comment itself.  It's fine for JSDoc to squeeze out the whitespace since it is using its own formatting rules.  What we're talking about here is the source text, which should read as it was written.

In general, anything that automatically rewrites my code without giving me the choice to accept or reject its changes gets a veto from me.

Reply:
Is there an option to have the reformatter insert these hyphens consistently everywhere? For me that would be a huge improvement. Without it, I also struggle to spot the transition from jsdoc declaration to prose about the declaration.

@FUDCo You had the same initial reaction to prettier doing a canonical reformatting of your source code. Interesting to explore how this case is similar and different.

Reply:
Huh. I thought I remembered that it did not. But I just looked at my own vscode IDE and I see that it does. So I agree. Thanks!